### ISSUE TITLE
Route caching with Laravel 5

### ISSUE THREAD

**Issue by sdebacker** [freekmurze/laravel-glide#19]
Does this package work with Laravel 5 route caching ?
I see that there is a closure on router->get…
```php
$this->app['router']->get($glideConfig['baseURL'].'/{all}', function () use ($glideConfig) {
…
})
```

**Comment by freekmurze** [comment]
From the [documentation](http://laravel.com/docs/5.0/controllers#route-caching): "If your application is exclusively using controller routes, you may take advantage of Laravel's route cache".

Wish I knew that a few months ago :-)

I'll replace the closure by a controller in the future. If you need this change soon, feel free to submit a pull request.

**Comment by freekmurze** [comment]
Thanks to @MatthiasDeWinter this is fixed in version 2.2.0
